molgenis-core-ui/src/main/java/org/molgenis/ui/controller/FeedbackController.java

```diff
@@ -51,17 +51,17 @@
 	private final UserService userService;
 	private final AppSettings appSettings;
 	private final CaptchaService captchaService;
-	private final JavaMailSender mailSender;
+	private final Supplier<MailSender> mailSenderSupplier;
```

Comment:
What do you think about hiding the Supplier behind our own JavaMailSender implementation?
- Easier to use, easier to change
- Requires less changes in existing code
- Expose JavaMailSender bean instead of MailSender such that we can build messages with attachments, inline resources etc.